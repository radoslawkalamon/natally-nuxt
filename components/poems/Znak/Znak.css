.znak {
  text-align: center;
}

.znak__image {
  display: block;
  margin: 0 auto;
  max-width: 280px;
  width: 100%;
}
