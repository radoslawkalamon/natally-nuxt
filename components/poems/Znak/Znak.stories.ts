import type { Meta, Story } from '@storybook/vue'
import PoemsZnak from '@/components/poems/Znak/Znak.vue'
import WrappersTextPoems from '@/components/wrappers/TextPoems/TextPoems.vue'

const meta: Meta = {
  title: 'Poems / Znak',
  component: PoemsZnak
}
export default meta

export const Default: Story = () => ({
  components: {
    PoemsZnak,
    WrappersTextPoems
  },
  template: `<div class="__storybook-section__">
    <WrappersTextPoems>
      <PoemsZnak />
    </WrappersTextPoems>
  </div>`
})
