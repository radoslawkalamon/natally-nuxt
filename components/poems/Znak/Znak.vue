<template>
  <div
    class="znak"
    data-test="poems-znak"
  >
    <p>Czekasz na znak?</p>
    <p>Oto on!</p>
    <p>
      <img
        alt="Stop!"
        class="znak__image"
        src="./images/znak.svg"
      >
    </p>
  </div>
</template>

<script src="./Znak.ts" lang="ts"></script>
<style src="./Znak.css" scoped></style>
