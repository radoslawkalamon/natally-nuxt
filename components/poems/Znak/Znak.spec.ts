import Znak from './Znak.vue'
import { shallRender } from '@/devtools/jest.shared.spec'

describe('Poems / Znak', () => {
  shallRender(Znak)
})
