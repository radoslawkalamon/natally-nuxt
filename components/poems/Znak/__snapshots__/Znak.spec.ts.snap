// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Poems / Znak Unit shall pass unit sanity test 1`] = `
"<div data-test=\\"poems-znak\\" class=\\"znak\\">
  <p>Czekasz na znak?</p>
  <p>Oto on!</p>
  <p><img alt=\\"Stop!\\" src=\\"./images/znak.svg\\" class=\\"znak__image\\"></p>
</div>"
`;
